README.md

```diff
@@ -1,27 +1,33 @@
-# The Rust Programming Language
-
-[![Rust Community](https://img.shields.io/badge/Rust_Community%20-Join_us-brightgreen?style=plastic&logo=rust)](https://www.rust-lang.org/community)
+<p align="center">
+  <img alt="The Rust Programming Language: A language empowering everyone to build reliable and efficient software" src="https://www.rust-lang.org/static/images/rust-social-wide.jpg">
+  <br>
+  <a href="https://www.rust-lang.org/">Website</a> |
+  <a href="https://www.rust-lang.org/learn/get-started">Getting started</a> |
+  <a href="https://www.rust-lang.org/learn">Learn</a> |
+  <a href="https://www.rust-lang.org/learn#learn-use">Documentation</a> |
+  <a href="CONTRIBUTING.md">Contributing</a>
+</p>
 
 This is the main source code repository for [Rust]. It contains the compiler,
 standard library, and documentation.
 
 [Rust]: https://www.rust-lang.org/
 
-**Note: this README is for _users_ rather than _contributors_.**
-If you wish to _contribute_ to the compiler, you should read
-[CONTRIBUTING.md](CONTRIBUTING.md) instead.
+## Why Rust?
```

Comment:
i'm also not sure whether it's worth having this. i don't expect that people visiting this repository need to be convinced that rust is good, i would expect that people coming to the repository are rust users already. maybe i'm wrong though.

Reply:
maybe, maybe not; I would expect to have at least some people who don't know Rust at all, to come to our README first before anything else, and giving them (without indirection) reason to like the language is IMO only positive

Reply:
I don't disagree with _having_ the Why but we can keep it very terse and make fewer unsubstantiated / overly-subjective claims. Some people I've met claim Rust _doesn't_ have great documentation, because they are used to ecosystems which are even more thoroughly documented (though I disagree about "great"). However we do have e.g. integrated doc tooling.